docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  backend_dummy:
+    container_name: backendDummy
+    build:
+      context: ./srcs/backend
+    ports:
+      - 8000:8000
+    # command: python manage.py runserver
+    volumes:
+      - ./srcs/backend/beePong:/usr/src/backend/beePong
+      # - ./tests:/code/tests
+  # db:
+  #   image: postgres:16
+  #   volumes:
+  #     - postgres_data:/var/lib/postgresql/data/
+  # nginx: 
+  #   container_name: nginx
+  #   depends_on:
+  #     - backendDummy
+  #   build:
+  #     context: ./srcs/nginx
+  #   env_file:
+  #     - .env
+  #   volumes:
+  #     - .srcs/backend/beePong:/beePong/app
+  #   ports:
+  #     - 127.0.0.1:8000:8000
+  #   restart: on-failure
```

Comment:
Need a server for frontend?

Reply:
We need Nginx as a web server and reverse proxy, also ssl_certificate in nginx conf file, so that frontend/api calls backend. But will setup later